src/olympia/editors/templates/editors/includes/files_view.html

```diff
@@ -1,5 +1,6 @@
 <div><strong>{{ _('Files in this version:') }}</strong></div>
 <ul>
+{% set latest_version=addon.find_latest_version(channel=amo.RELEASE_CHANNEL_LISTED) %}
```

Comment:
yeah, or you might have to move it inside the `{% for file ... %}` loop and do `channel=file.version.channel`, I can't remember how its included in other templates.